services/static-webserver/client/source/class/osparc/dashboard/ResourceMoreOptions.js

```diff
@@ -160,10 +190,39 @@ qx.Class.define("osparc.dashboard.ResourceMoreOptions", {
             this.fireDataEvent("openService", this.__resourceData);
             break;
         }
-      });
-      toolbar.add(openButton);
+      }, true);
+    },
 
-      this._add(toolbar);
+    __confirmUpdate: function() {
+      this.__openButton.setLabel("Update");
+      this.__openButton.addListenerOnce("tap", () => {
+        const msg = this.tr("Some of your services are outdated. For imploved performance please update to the latest version.");
```

Comment:
There's a typo here "imploved"

Reply:
@elisabettai I improved the text with.
Some of your services are outdated. For imploved performance please update to the latest version.
What do you think?